src/conponents/Header/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import cn from 'classnames';
+import React, { FormEvent } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  handleAddTodo: (event: FormEvent<HTMLFormElement>) => void;
```

Comment:
Why pass all event? This is the internal detail of the component.
Better variant is

```suggestion
  handleAddTodo: (todoText: string) => void;
```